ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -770,6 +770,30 @@ FCU_DFMSS_DSP_ZHM_RTM_CHWZC_CHWDT:
   - CHWZC
   - CHWDT
 
+FCU_DFSS_STM_RTC_RHM:
+  description: "CRAC unit cooling and humidification with compressor and alarm monitoring."
+  is_canonical: true
+  implements:
+  - FCU
+  - DFSS
+  - STM
+  - RTC
+  - RHM
+  opt_uses:
+  - compressor_run_time_accumulator
+  - humidifier_run_time_accumulator
```

Comment:
if it has a humidifier it belongs on an humidifier type (which we have with types like RHDHC).

Reply:
Moved to RHC